packages/test-support/deno.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@commontools/test-support",
```

Comment:
Check out [`@std/testing/snapshot`](https://jsr.io/@std/testing/doc/snapshot) -- I've seen it used by [Astral](https://github.com/lino-levan/astral), which provides the ability to run tests and store outputs so that future tests are compared to it (not dissimilar to the golden generations here), maybe this could be used for fixture generation, or at least some ideas could be used